### PR TITLE
Add global field to react tokens

### PR DESCRIPTION
**Pull request by jschuler**
This adds the global var that is referenced to get the final value.

Example:
```
export default {
  name: '--pf-c-alert--BackgroundColor',
  value: '#fff',
  var: 'var(--pf-c-alert--BackgroundColor)',
  global: 'var(--pf-global--BackgroundColor--light-100)' // this field is new
};
```

**What**: Towards https://github.com/patternfly/patternfly-react/issues/3521

**Additional issues**:

### CODE REVIEW

File: packages/patternfly-4/react-tokens/src/generateTokens.js [hunk 1]
```diff
@@ -37,11 +37,17 @@ cssFiles.forEach(filePath => {
         });
         // Avoid stringifying numeric chart values
         const chartNum = decl.property.startsWith('--pf-chart-') && !isNaN(populatedValue);
```
Comment (redallen) [inline packages/patternfly-4/react-tokens/src/generateTokens.js]: Can't we avoid stringifying all values and remove this line?

File: packages/patternfly-4/react-tokens/src/generateTokens.js [hunk 2]
```diff
@@ -37,11 +37,17 @@ cssFiles.forEach(filePath => {
...
+        const calculatedValue = chartNum ? Number(populatedValue).valueOf() : populatedValue;
```
Comment (redallen) [inline packages/patternfly-4/react-tokens/src/generateTokens.js]: Can we move this logic to be on line 36?

File: packages/patternfly-4/react-tokens/src/generateTokens.js [hunk 3]
```diff
@@ -37,11 +37,17 @@ cssFiles.forEach(filePath => {
...
         tokens[key] = {
+          ...tokens[key],
           name: property,
-          value: chartNum ? Number(populatedValue).valueOf() : populatedValue,
+          value: calculatedValue,
           var: `var(${property})`
         };
+        const isGlobal = populatedValue.indexOf('--pf-global') > -1;
```
Comment (redallen) [inline packages/patternfly-4/react-tokens/src/generateTokens.js]: Nit: You can use `.includes()`  and spread into `tokens[key] = ` to reduce these 4 lines down to 1. Like
```js
tokens[key ] = {
  ...(populatedValue.includes('--pf-global') && { global: populatedValue })
}
```

**Comment by jschuler** [comment]
CLosed in favor of #3896

**Pull request closed without merging**